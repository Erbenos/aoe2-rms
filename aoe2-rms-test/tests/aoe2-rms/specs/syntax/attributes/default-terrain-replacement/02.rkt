#lang aoe2-rms
; Replace everything with ice to see which routes the connections are taking.

<CONNECTION-GENERATION>
(create-connect-all-lands
 (default-terrain-replacement 'ICE))
