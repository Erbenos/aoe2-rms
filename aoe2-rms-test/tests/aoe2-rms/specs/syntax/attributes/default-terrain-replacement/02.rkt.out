<CONNECTION_GENERATION>
create_connect_all_lands {
  default_terrain_replacement ICE
}
